java/change_atmosphere_effect/README.md

```diff
@@ -0,0 +1,29 @@
+# Change Atmosphere Effect
+Change the appearance of the atmosphere in a scene.
+
+![Change_Atmosphere_Effect](change_atmosphere.gif)
```

Comment:
This should be of the format:

![Change Atmosphere Effect App](change-atmosphere-effect.png)